ethereum_history_api/circuits/lib/src/misc/arrays_test.nr

```diff
@@ -151,3 +151,43 @@ mod resize {
         let _ = array_equals(resize([1, 2]), [1]);
     }
 }
+
+mod subarray {
+    use crate::misc::arrays::subarray;
+
+    #[test]
+    fn success() {
+        let array = [1, 2, 3, 4, 5];
+        assert_eq(subarray(array, 0), array);
```

Comment:
Without a more descriptive name of `subarray` function, it's very unclear what is going on in this test

Reply:
I cannot find a better name for this function...